lib/Number/Compare.pm

```diff
@@ -1,8 +1,10 @@
 package Number::Compare;
 use strict;
+use warnings;
+
 use Carp qw(croak);
-use vars qw/$VERSION/;
-$VERSION = '0.03';
+
```

Comment:
I suppose at this point we're not really excluding people by requiring 5.6 and later;  I wonder if we should signal that more clearly in the changelog

Reply:
I'll add a line to that effect